lemur/plugins/lemur_aws/plugin.py

```diff
@@ -325,13 +321,9 @@ def get_load_balancers(self, options, **kwargs):
                     capture_exception()
                     continue
 
+            continue
```

Comment:
not sure if `continue` here is intentional, since the remainder of the code would never be reached

Reply:
Thanks for finding this bit of debugging. To catch something similar in the future I added a unit test to make sure plugin is always returning both types of ELBs.